mmcv/ops/contour_expand.py

```diff
@@ -33,6 +33,18 @@ def contour_expand(kernel_mask, internal_kernel_label, min_kernel_area,
     if isinstance(internal_kernel_label, np.ndarray):
         internal_kernel_label = torch.from_numpy(internal_kernel_label)
 
-    label = ext_module.contour_expand(kernel_mask, internal_kernel_label,
-                                      min_kernel_area, kernel_num)
+    if torch.__version__ == 'parrots':
+        if kernel_mask.shape[0] == 0 or internal_kernel_label.shape[0] == 0:
+            label = []
+        else:
+            label = ext_module.contour_expand(
+                kernel_mask,
+                internal_kernel_label,
+                min_kernel_area=min_kernel_area,
+                kernel_num=kernel_num)
+            label = label.tolist()
+        label = [np.array(m, dtype=np.int) for m in label]
```

Comment:
Should we return ndarray or a tensor here? It does not seem to be matched with the docstring and the return type of label when using pytorch.